cmd/embedded-cluster/network.go

```diff
@@ -50,3 +50,11 @@ func DeterminePodAndServiceCIDRs(c *cli.Context) (string, string, error) {
 	}
 	return netutils.SplitNetworkCIDR(c.String("cidr"))
 }
+
+// getCIDRs returns the CIDRs in use based on the provided cli flags.
+func getCIDRs(c *cli.Context) (string, string, string) {
+	if c.IsSet("pod-cidr") && c.IsSet("service-cidr") {
```

Comment:
```suggestion
	if c.IsSet("pod-cidr") || c.IsSet("service-cidr") {
```

Reply:
this needs to be an `||` as well: https://github.com/replicatedhq/embedded-cluster/blob/main/cmd/embedded-cluster/network.go#L48

Reply:
Done via - https://github.com/replicatedhq/embedded-cluster/pull/1355/commits/106fee65e04a20a6c8fbd256ae9e8a1bd29a402b